packaging/ansible-runner-service-project/project/roles/hc-gluster-cgroups/tasks/main.yml

```diff
@@ -1,26 +1,27 @@
 ---
 - name: Create systemd overrides directory for glusterd
-  file:
+  ansible.builtin.file:
     path: /etc/systemd/system/glusterd.service.d/
     state: directory
+    mode: 0755
 
 - name: Create glusterd cgroups CPU configuration file
-  copy:
+  ansible.builtin.copy:
     src: 99-cpu.conf
     dest: /etc/systemd/system/glusterd.service.d/99-cpu.conf
     mode: preserve
```

Comment:
@gobindadas What should be the actual permissions? Copying permissions from local directory seems unreliable/dangerous to me

Reply:
The actual permission is 644

Reply:
fixed